modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalAddListTypeDefinition.tsx

```diff
@@ -31,10 +31,16 @@ const ModalAddListTypeDefinition: React.FC<IProps> = ({
 
 	const onSubmit = async (values: TInitialValues) => {
 		try {
-			await API.save({item: values, method: 'POST', url: apiURL});
+			await API.save({
+				item: values,
+				method: 'POST',
+				returnValue: true,
+				url: apiURL,
+			});
 
 			onClose();
-			window.location.reload();
+
+			setTimeout(() => window.location.reload(), 1000);
```

Comment:
Adding time outs should be avoided, specially considering that this one is being added to make a test work. Lets look for a different solution.